packages/sdk/package.json

```diff
@@ -185,14 +186,14 @@
     }
   },
   "scripts": {
-    "build": "vite build && tsc -p ./tsconfig.build.json --emitDeclarationOnly --declaration --declarationMap",
+    "build": "mkdir -p dist && vite build && tsc -p ./tsconfig.build.json --emitDeclarationOnly --declaration --declarationMap",
     "typecheck": "tsc -p ./tsconfig.build.json --noEmit",
     "lint:ci": "npx biome ci",
     "lint": "npx biome check",
     "bench": "vitest bench",
     "bench:ci": "CI=true vitest bench",
-    "clean": "rm -rf dist",
-    "dev": "vite build --watch",
+    "clean": "mkdir -p dist && rm -rf dist",
```

Comment:
Can you explain why we care to create before we destroy here?

Reply:
lol i have no recollection of this